official/mnist/mnist.py

```diff
@@ -165,7 +153,8 @@ def train_input_fn():
 
   # Evaluate the model and print results
   def eval_input_fn():
-    return eval_dataset(FLAGS.data_dir).make_one_shot_iterator().get_next()
+    return dataset.test(FLAGS.data_dir).batch(
+        FLAGS.batch_size).make_one_shot_iterator().get_next()
```

Comment:
nit: I don't think this is the right code style. Can we split this onto two lines instead?

Reply:
This was the result of running the Python formatter (https://github.com/google/yapf), so it should be right? :)

Reply:
That's surprising; the part that seemed weird to me was indenting the arguments to the next line and then following up with more function calls. If the formatter says it's good though, should be fine.